### DISCUSSION TITLE
Nginx Ingress like Granafa Dashboard

### DISCUSSION THREAD

**Discussion by BrikerMan** [Kong/kong#8000] [Ideas and feature requests]
I used to use the Nginx Controller with [Grafana Nginx Ingress Dashboard](https://grafana.com/grafana/dashboards/9614), which is really nice.

And I tried out the official Kong Grafana dashboard, it missing a lot of features, for example.
1. Stacked HTTP-code / service view.
2. Success rate view.
3. Request by service view
4. Request latency by service view.

Are we going to have view like that or just be able to access to the nginx metrics?

**Comment by juzhiyuan** [comment]
cc @liuxiran @tokers to have a look also 😄

**Comment by kikito** [comment]
Hi, thank you for your report. We gradually improve all our plugins, but our resources aren't unlimited and we must prioritize.

If you have a specific need that you need Kong to support, may I suggest opening a pull request? The code for the Prometheus plugin is currently located in https://github.com/Kong/kong/tree/master/kong/plugins/prometheus.

**Reply by BrikerMan** [reply]
Yep, I am using this one. Can we have direct access to Nginx Metrics? There are more pre-build dashboards for Nginx matrics.
